actions-server/package.json

```diff
@@ -4,7 +4,11 @@
   "description": "",
   "main": "src/index.js",
   "scripts": {
-    "start": "nodemon -r esm src/server.js"
+    "build": "tsc",
+    "start": "node build/actions-server/src/server.js",
+    "start-debug": "nodemon --inspect",
+    "lint": "tsc --noEmit && yarn run prettier",
```

Comment:
`tsc` is still failing when dependencies are _not_ installed in `/app` due to the generated `graphql.tsx`. I tried adding what I thought would have been the required dev dependencies to `actions-server`:

* `@graphql-codegen/typescript`
* `@apollo/react-hooks`

but that still didn't seem to do the trick 🤔

Reply:
See https://github.com/avimoondra/fishbowl/pull/82 for a possible solution.

Reply:
Integrated into here, but without the react bits, see https://github.com/avimoondra/fishbowl/pull/74/commits/98805f1036d31dbd86c6a4f53d26e140aa380f64#diff-48cd5bd40869415ec9c4406f3b723d9aR15